### ISSUE TITLE
GNU/Linux on arm platform works

### ISSUE THREAD

**Issue by Sleeplessy** [bendahl/uinput#7]
Tested on a self-made board with a personal hacked kernel (though i haven't edit the udev part),cross-compiling using the official go compiler with GOARCH=arm,it works perfect.But it seems to be blocked for about 3 seconds if send key events through a muiltithreaded client...need more test huh.

**Comment by Sleeplessy** [comment]
The cpu is a cortex-a9 so i guess the architecture is ARMv7-A.

**Comment by bendahl** [comment]
That's great news! I will update the README.md accordingly. Do you have an update on the blocking issue? Is this something that we need to look into? If so, it would be great to have some example code in order to reproduce the issue. I'll leave the issue open in order to track this. Let me know if you've got any news.

**Comment by Sleeplessy** [comment]
I'm not sure whether it is caused by my code or it is the a system leak due to the cheap hardware  or maybe it is due to this library. I'm still testing,once i got a result i'll reply here.

**Comment by bendahl** [comment]
Are there any news regarding this? Can you provide more details on the matter? Maybe a code sample?

**Comment by bendahl** [comment]
I did some further testing and so far I couldn't identify any issues. In order to test the multithreaded use of the keyboard device, I did create a simple REST client with Gin Gonic that expects a character as the sole parameter to a GET request. This is then mapped to a keypress. For testing purposes I built a java client that issues a couple of hundreds of requests from different threads. In order to ensure that there's no issue related to goroutine scheduling on the server side, I did test with GOMAXPROCS set to the default (number of CPUs) as well as set to 1. Both setups did work fine and the response times were almost all sub-millisecond. I also verified that the letters were printed on the screen. This also worked. 

In conclusion, I will close this issue for now as I don't see any issue related to this library. Feel free to reopen the issue in case you should discover a remaining problem that I might be overlooking. In case you do, it would also be a great help if you could provide some more information (maybe a simple test case) in order to narrow down the possible causes.
